packages/@okta/vuepress-site/code/android/index.md

```diff
@@ -26,30 +26,26 @@ These resources walk you through adding user authentication to your Android app
 	</li>
 </ul>
 
-## Integrate with Okta using the Okta-hosted Sign-In Widget
+## Add sign-in with Okta to your app
 
-These SDKs help you integrate with Okta by redirecting to the Okta Sign-In Widget using OpenID Connect (OIDC) client libraries.
+There are two ways to integrate sign-in with Okta to your app. The easiest to integrate is the redirect model which uses the Okta-hosted Sign-In Widget. You open the widget, Okta processes the sign-in attempt, and sends the result to your app. The other is the embedded model, where you manage each step in the sign-in flow. See [Overview of the mobile Identity Engine SDK](/docs/guides/mobile-idx-sdk-overview/ios/main/) for more detail.
```

Comment:
This is the `android` markdown file, but the link to the IDX docs references the iOS version.

Reply:
Done, fixed in next commit.